website/versioned_docs/version-v18.5.0/troubleshooting/troubleshoot_database_migration.md

```diff
@@ -0,0 +1,39 @@
+---
+id: troubleshooting-database-migrations
+title: Troubleshooting Database Migrations
+hide_title: false
+---
+
+This document provides guidance on how to troubleshoot database migration when trying to upgrade to v18.5.1.
+
+:::info
+
+This document is specifically intended to provide a potential resolution for issues with the database migration to v18.5.1+.
+The advice in this document may be used for migrations to other versions, but please be mindful they may not provide the same resolution.
+:::
+
+## Failing Database Migrations
+
+When trying to deploy the v18.5.1 of Cumulus, specifically the `data-persistence-tf` module, there is a chance the terraform deployment task times out. The timeout usually occurs when the `<prefix>-postgres-db-migration` lambda is invoked. It will attempt to run the migrations, but the lambda reports that the table is locked. This occurs when the lambda is trying to run a migration that is NOT in the `Migrations Table`.
```

Comment:
```suggestion
When trying to deploy the v18.5.1 of Cumulus, specifically the `data-persistence-tf` module, there is a chance the terraform deployment task times out. The timeout usually occurs when the `<prefix>-postgres-db-migration` lambda is invoked. It will attempt to run the migrations, which may take longer than the Lambda's timeout allows. This occurs when the lambda is trying to run a new migration that has not already been applied (e.g. is NOT in the PostgreSQL `Migrations Table`).
```